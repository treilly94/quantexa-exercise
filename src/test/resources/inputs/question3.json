{"transactionId":"01","accountId":"10","transactionDay":1,"category":"AA","transactionAmount":1.00},
{"transactionId":"02","accountId":"10","transactionDay":2,"category":"AA","transactionAmount":1.00},
{"transactionId":"03","accountId":"10","transactionDay":3,"category":"AA","transactionAmount":1.00},
{"transactionId":"04","accountId":"10","transactionDay":4,"category":"AA","transactionAmount":1.00},
{"transactionId":"05","accountId":"10","transactionDay":5,"category":"AA","transactionAmount":1.00},
{"transactionId":"06","accountId":"20","transactionDay":5,"category":"BB","transactionAmount":1.11},
{"transactionId":"07","accountId":"20","transactionDay":6,"category":"FF","transactionAmount":1.22},
{"transactionId":"08","accountId":"20","transactionDay":7,"category":"AA","transactionAmount":1.33},
{"transactionId":"09","accountId":"20","transactionDay":8,"category":"CC","transactionAmount":1.44},
{"transactionId":"10","accountId":"20","transactionDay":9,"category":"CC","transactionAmount":1.55},
{"transactionId":"11","accountId":"30","transactionDay":4,"category":"CC","transactionAmount":5.00},
{"transactionId":"12","accountId":"30","transactionDay":5,"category":"AA","transactionAmount":4.00},
{"transactionId":"13","accountId":"30","transactionDay":6,"category":"FF","transactionAmount":3.00},
{"transactionId":"14","accountId":"30","transactionDay":7,"category":"SS","transactionAmount":2.00},
{"transactionId":"15","accountId":"30","transactionDay":8,"category":"KK","transactionAmount":1.00}
