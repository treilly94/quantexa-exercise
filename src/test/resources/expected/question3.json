{"transactionDay":1,"AccountId", "Maximum":1.0, "Average", "AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":2,"AccountId", "Maximum":1.3, "Average", "AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":3,"AccountId", "Maximum":3.0, "Average", "AA Total Value", "CC Total Value", "FF Total Value"}

{"transactionDay":1,"accountId":"10","Maximum":1.00,"Average":1.000,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":2,"accountId":"10","Maximum":1.00,"Average":1.000,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":3,"accountId":"10","Maximum":1.00,"Average":1.000,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":4,"accountId":"10","Maximum":1.00,"Average":1.000,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":5,"accountId":"10","Maximum":1.00,"Average":1.000,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":5,"accountId":"20","Maximum":1.11,"Average":1.110,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":6,"accountId":"20","Maximum":1.22,"Average":1.165,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":7,"accountId":"20","Maximum":1.33,"Average":1.220,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":8,"accountId":"20","Maximum":1.44,"Average":1.275,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":9,"accountId":"20","Maximum":1.55,"Average":1.330,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":4,"accountId":"30","Maximum":5.00,"Average":5.000,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":5,"accountId":"30","Maximum":5.00,"Average":4.500,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":6,"accountId":"30","Maximum":5.00,"Average":4.000,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":7,"accountId":"30","Maximum":5.00,"Average":3.500,"AA Total Value", "CC Total Value", "FF Total Value"},
{"transactionDay":8,"accountId":"30","Maximum":5.00,"Average":3.000,"AA Total Value", "CC Total Value", "FF Total Value"}